wiki/osu!_File_Formats/Osu_(file_format)/en.md

```diff
@@ -80,7 +80,7 @@ These options are only relevant when opening maps in the [beatmap editor](/wiki/
 | `CircleSize` | Decimal | CS setting (0–10) |
 | `OverallDifficulty` | Decimal | OD setting (0–10) |
 | `ApproachRate` | Decimal | AR setting (0–10) |
-| `SliderMultiplier` | Decimal | Base slider velocity in hecto-[osu! pixels](/wiki/osupixel) per beat <!-- TODO: this prefix sounds awful ಠ_ಠ --> |
+| `SliderMultiplier` | Decimal | Base slider velocity in hundred-[osu! pixels](/wiki/osupixel) per beat |
```

Comment:
`hecto` works as a prefix (akin to `kilo` in *kilogram*), `hundred` doesn't, so you need a very small rephrase here
```suggestion
| `SliderMultiplier` | Decimal | Base slider velocity in hundreds of [osu! pixels](/wiki/osupixel) per beat |
```

Reply:
i got one better https://github.com/ppy/osu-wiki/pull/6485#discussion_r762086695

Reply:
I actually like it less!

Reply:
So which one should it use? `hundreds of` or `100 `?
(personally I like `hundreds of` because the original was a prefix)

Reply:
we've had a small debate over it in Discord, and came to a conclusion that "... in 100 osu! pixels per beat" is a valid construction.
meanwhile, I've looked it up, and it seems you can still measure something in hundreds, or dozens, or thousands of something else: https://www.google.com/search?q=%22distance+in+hundreds+of+meters%22
so.. idk?

Reply:
Okay I reckon just go with this one, because its x hundreds of osu! pixels